File: first-contributions/sphere-volume/SphereVolume.java
import java.text.DecimalFormat;

public class SphereVolume{
	private static final DecimalFormat df = new DecimalFormat("0.00");

	public static void main(String[] args){
		// 4/3 will mess up the calculation.
		// 4.0/3.0 gives correct answer
		double volume = Math.pow(8,3) * Math.PI * (4.0/3.0);
		System.out.println("Volume: " + df.format(volume));
	}	
}

File: first-contributions/sphere-volume/SphereVolume.c
#include <stdio.h>
#include <math.h>

int main(){
	float volume = (4.0/3.0)*M_PI*8*8*8;
	float volumeRounded = (volume*100)/100;
	printf("\nVolume: %.2f\n",volumeRounded);
	return 0;
}
